dev/benchmarks/platform_views_layout/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
```

Comment:
Do you need to check in this file, or will flutter run update it correctly?

Reply:
If CocoaPods wants to touch the Xcode project, imo I think we should check it in so there's not dirty state every time someone runs the benchmark locally.